packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -59,6 +56,47 @@ describe('HasMany relation', () => {
     expect(persisted.toObject()).to.deepEqual(order.toObject());
   });
 
+  it('can patch many instances', async () => {
+    await givenCustomerOrder({description: 'order 1', isDelivered: false});
+    await givenCustomerOrder({description: 'order 2', isDelivered: false});
+    const patchObject = {description: 'new order'};
```

Comment:
I'm being picky here, but can we patch the `isDelivered` property to be true and keep the descriptions as it? Feel free to ignore.

Reply:
ah that's a more practical use case 👍